services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -208,6 +208,14 @@ class DynamicSidecarSettings(BaseCustomSettings):
         ),
     )
 
+    DYNAMIC_SIDECAR_ATTACH_DETACH_NETWORK: PositiveFloat = Field(
+        3.0 * MINS,
+        description=(
+            "when attaching a network to a container or when removing "
```

Comment:
Can you rename the field so it suggests a name. Add units (secs?) as well at least in the description